README.md

```diff
@@ -84,15 +84,22 @@ After **the issue** is resolved and merged, `hotfix` branch will be deleted and
 ## Github, Gitlab(s), etc?
 
 We use github for common activities, as it is currently most easily discoverable space.
-**UNIGE Platform is deployed with CI/CD from integral gitlab**, which is pushing changes with github.
-It is also used for internal astro-only projects.
+Hence github hosts the **"community version"** of the software.
+
+**UNIGE MMODA Site is deployed with CI/CD from integral gitlab**, which is pushing changes with github.
```

Comment:
Are you here talking about the automatic mirroring of any push in gitlab towards github?

Reply:
this was there before actually and it was not at all clear. I changed.